### PR TITLE
UW Cloud Directory changes for skeleton

### PR DESCRIPTION
**Pull request by goulter**
Contains styles, header, footer and form from outgoing directory. 
Modularized html code for easier modification. 
**Search functionality will not work at this time, Form does not call search service**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.idea
+glenrg.crt
```
Comment (tomthorogood) [inline .gitignore:2]: Please don't include personal files here. Instead, set up a global .gitignore file on your machine to catch these things. 

```
echo "glenrg.crt" > ~/.gitignore_global
git config --global core.excludesfile ~/.gitignore_global
```

Then, you don't have to have commits for your own personal files in the history.

File: husky_directory/static/css/mini.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+@font-face {
+  font-family: 'Encode Sans Compressed';
+  src: url('../assets/fonts/encodesanscompressed-600-semibold-webfont.eot');
+  src: url('../assets/fonts/encodesanscompressed-600-semibold-webfont.eot?#iefix') format('embedded-opentype'), url('../assets/fonts/encodesanscompressed-600-semibold-webfont.woff') format('woff'), url('../assets/fonts/encodesanscompressed-600-semibold-webfont.ttf') format('truetype'), url('../assets/fonts/encodesanscompressed-600-semibold-webfont.svg#encodesanscompressed-600-semibold-webfont') format('svg');
+  font-weight: 600;
+  font-style: normal;
+}
+
+/* Pulled from our CSS */
```
Comment (tomthorogood) [inline husky_directory/static/css/mini.css:11]: What is "our"? Can you link to it?

File: husky_directory/static/css/print.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/****** Print Styles ******/
+
+#wheader, .cblock, .rsummary, .mobileHeader, #footerMain, #footerMainNoPatch, .title, .btn-primary  {
+	display: none;
+}
```
Comment (tomthorogood) [inline husky_directory/static/css/print.css:5]: Please add a blank line at the end of this file (there are several instances of this; you can see the red warning circle in this github view)

File: husky_directory/static/css/structure.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
+/* Not in our normal CSS; custom */
```
Comment (tomthorogood) [inline husky_directory/static/css/structure.css:1]: What is "our normal CSS"?

File: husky_directory/templates/index.html [hunk 5]
```diff
@@ -1,11 +1,100 @@
...
+
+        <link rel="stylesheet" type="text/css" media="print" href="{{ url_for('static', filename='css/print.css') }}">
+
+        <style>
+         .btn.search:hover { color: black; }
+        /*------------ Workday changes --------- */
+        ul.dir-listing { list-style-type: none; padding: 0; }
+        ul.multiaddr, ul.title, ul.address { list-style-type: none; padding: 0; }
+        li.dir-boxstuff { padding-bottom: .6em }
+        /*------------ End Workday stuff-------*/
```
Comment (tomthorogood) [inline husky_directory/templates/index.html:28]: Can you explain why these changes are required? (Or was this code copied from elsehwere?)

**Review by tomthorogood — CHANGES_REQUESTED** [review]
Looks pretty good, just some minor changes requested.

File: .gitignore (hunk 1 above)
Comment (tomthorogood) [reply .gitignore:2]: NB: I was only refering to glenrg.crt; .idea is pretty shared, so can stay :)

File: husky_directory/static/css/mini.css (hunk 2 above)
Comment (goulter) [reply husky_directory/static/css/mini.css:11]: I pulled in mini.css, the comment is pre-existing

File: husky_directory/templates/index.html (hunk 5 above)
Comment (goulter) [reply husky_directory/templates/index.html:28]: This is pre-existing code from the index.html in the old uw directory.

File: husky_directory/static/css/structure.css (hunk 4 above)
Comment (goulter) [reply husky_directory/static/css/structure.css:1]: I pulled in structure.css, the comment is pre-existing

**Review by tomthorogood — APPROVED** [review]
